decidim/decidim
# Bubble jQuery events with the custom confirm dialog

#### :tophat: What? Why?
The confirm dialog does not bubble jQuery events because they are not type of Event.

You could see the following exception in the developer console:

```
TypeError: Right-hand side of 'instanceof' is not callable
```

Originates from this line:
https://github.com/decidim/decidim/blob/69204664eb115a6d7ffd8ac8f2a3d9fc787ae500/decidim-core/app/assets/javascripts/decidim/confirm.js.es6#L118

#### :pushpin: Related Issues
- Related to #6253

#### Testing
- Create a new button e.g. `<button data-confirm="Are you sure?">Test</button>`
- Click the button
- Click "OK" in the confirm dialog
- Check the developer console

This is because the event that this script tried to bubble was a `jQuery.Event`. Fortunately, the original event is stored in this object's `originalEvent` property which can be used to fix it.

#### :clipboard: Checklist
